mne/tests/test_epochs.py

```diff
@@ -3254,6 +3254,22 @@ def test_make_fixed_length_epochs():
     assert len(epochs_annot) > 10
     assert len(epochs) > len(epochs_annot)
 
+    # overlaps
+    epochs = make_fixed_length_epochs(raw, duration=1, preload=True)
```

Comment:
```suggestion
    epochs = make_fixed_length_epochs(raw, duration=1)
```

Reply:
I'll comment here bc it is the same for the suggestions before. If I do not specify the preload, the `assert len(epochs)` fails because the len(epochs) seem to only be working with preloaded data.

I thought I should keep it for this reason, what do you think? Could I change the check in a more optimized way?

Reply:
Edit: I changed it to check the length of the events in the epochs objects to avoid preloading but still performing an analogous check.

Now the tests are not anymore on the same style of the previous ones in the same file (`mne/tests/test_epochs.py`)  that create epochs with `preload=True` and asserting `len(epochs)> n`.

@agramfort do you think they should also be modified too for better readability and optimization?